source/docs/articles/wordpress/wordpress-and-pantheon-varnish-cache.md

```diff
@@ -0,0 +1,62 @@
+---
+title: WordPress and Pantheon's Varnish Servers
+description: Detailed information on understanding the interactions of WordPress Pantheon's Varnish edge caching servers.
+category:
+  - developing
+  - WordPress
+  - cache
+  - Varnish
+keywords: wordpress, cache, caching, varnish
+---
+While WordPress Core does come with object caching, it does not include any direct user interface (UI) to control that functionality. On Pantheon, all WordPress sites include a plugin called Pantheon Cache which not only includes a simple UI for administering the site cache, it also has programatic functionality for selectively clearing the Pantheon Varnish caching for individual pages and paths. In this article, we're going to go over some of the details about how these pieces all fit together.
+
+<h4>Pantheon Cache Administration</h4>
+In order to access the Pantheon Cache administration, log into your WordPress admin and navigate to /wp-admin/options-general.php?page=pantheon-cache. Once there, you will see the following:
+
+![pantheon wordpress cache admin](/source/docs/assets/images/wp-pantheon-cache-ui.png)
+
+The options are straight forward:
+
+1.) Set the page cache expiration time via the HTML TTL header. The default value is 600 seconds. For no page caching, set this to 0. This is not recommended for Live environments! Pantheon's Varnish servers obey the page TTL header. If it's set to 0, it will not cache the page.
```

Comment:
Have you tested 0 seconds? I don't think that works currently. I think current behavior is wonky, allowing minimum of 60 seconds, and if you try to set it to less than 60 seconds it will set it the TTL to 600 seconds. See current version https://github.com/pantheon-systems/WordPress/blob/6b1eefb5269a266cca3db8ceb6372899e844a41c/wp-content/mu-plugins/pantheon/pantheon-cache.php#L204

Work being done on release-4.4 branch that will change this behavior: https://github.com/pantheon-systems/WordPress/pull/34